torch/testing/_internal/distributed/_tensor/common_dtensor.py

```diff
@@ -78,6 +78,17 @@ def reset_parameters(self):
         self.net2.reset_parameters()
 
 
+class MLPStacked(nn.Module):
+    def __init__(self, device):
```

Comment:
nit: I think it would be nice to have a `num_layers` arg (can have a default) to control how many MLP layers this stacked MLP construct